### PR TITLE
Add compatibility with symfony 3

### PR DESCRIPTION
**Pull request by pane-e-design**

### CODE REVIEW

**Comment by nullziu** [comment]
``` "symfony/framework-bundle": "3.*" ```

This would drop compatibility with symfony 2.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (smatyas) [inline .gitignore]: I prefer using global gitignore for things like this.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (burci) [inline .gitignore]: This is not needed.

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     ],
     "require":     {
         "php":                      ">=5.3.1",
-        "symfony/framework-bundle": "2.*"
+        "symfony/framework-bundle": "3.*"
```
Comment (burci) [inline composer.json]: Change to ```"symfony/framework-bundle": ">=2.0.0"```

**Pull request merged**